math/mathcore/inc/Fit/FitUtil.h

```diff
@@ -311,7 +311,7 @@ namespace FitUtil {
   /**
       evaluate the Poisson LogL given a model function and the data at the point p.
       return also nPoints as the effective number of used points in the LogL evaluation
-      By default is extended, pass extedend to false if want to be not extended (MultiNomial)
+      By default is extended, pass extend to false if want to be not extended (MultiNomial)
```

Comment:
Here it should be: 

By default it is computed in extended mode. Set the parameter `extended` to false.....

Reply:
That's a re-phrasing. I suggest you do a separated PR for that because I noticed this comment comes in several places. I only did simple obvious typos fixes in that PR.